docs/how-to/run_upf_in_dpdk_mode.md

```diff
@@ -0,0 +1,125 @@
+# Run UPF in DPDK mode
+
+This guide covers how to deploy the User Plane Function (UPF) in DPDK mode using 
+the `sdcore-user-plane` Juju bundle.
+
+## Requirements
+
+- A Kubernetes cluster which meets below requirements:
+  - host CPU that supports AVX2, RDRAND and PDPE1GB instructions (Intel Haswell, AMD Excavator 
+    or equivalent)
+  - SR-IOV interfaces for Access and Core networks
+  - At least two 1G HugePages available
+  - `driverctl` installed
+  - LoadBalancer with 1 available address for the UPF
+  - Multus CNI enabled
+- Juju >= 3.1/stable
+- A Juju controller bootstrapped onto the Kubernetes cluster
+- Juju model created
+
+## Set up network interfaces
+
+As `root` user, load the `vfio-pci` driver on the Kubernetes host:
+
+```shell
+echo "vfio-pci" > /etc/modules-load.d/vfio-pci.conf
+modprobe vfio-pci
+```
+
+```{note}
+Using `vfio-pci`, by default, needs IOMMU to be enabled. In the environments which do not support
```

Comment:
What determines if `IOMMU` is available? (this is a genuine question, not a suggestion)

Reply:
Basically it depends on the hardware. Unfortunately, it's not like a CPU flag or something we could easily check from the code. 
[Here's a reference article](https://en.wikipedia.org/wiki/List_of_IOMMU-supporting_hardware)

Reply:
It looks like all the recent CPU's do support it though (?) I'd simply place this in the set of requirements with the others at the top of this file.

Reply:
Yes, it looks like it and yet most EC2 flavors don't support it (perhaps it's not enabled in BIOS on the underlying servers?). I'm not sure if we should make it a requirement. It could be a recommendation, but then again we'll need this note anyway, so does it make sense to mention it in the requirements in the first place?

Reply:
Ok I'm fine with keeping it as is then